analysers/analyser_osmosis_parking_highway.py

```diff
@@ -57,6 +60,47 @@
   highway.id IS NULL
 """
 
+
+sql20 = """
+SELECT
+  DISTINCT ON(parking.id)
+  parking.id,
+  private_parking_way.id,
+  ST_AsText(ST_Centroid(parking.linestring)),
+  private_parking_way.tags->'access',
+  parking.tags->'access'
+FROM
+  ways AS parking
+  JOIN park_highway AS private_parking_way ON
+    parking.linestring && private_parking_way.linestring AND
+    ST_Intersects(parking.linestring, private_parking_way.linestring) AND
+    private_parking_way.highway = 'service' AND
+    private_parking_way.tags?'access' AND
+    private_parking_way.tags->'access' IN ('private', 'permit', 'delivery', 'customers')
```

Comment:
Add `private_parking_way.tags != ''::hstore AND` to be able to use the index (more a good practice to add it every where, even if may not required here).

Reply:
I don't fully understand with what you mean by 'to be able to use the index'. (It'll always be true, since I'm using a subset of the table `highways`)

Reply:
Right!